stellarobservatory/quorums_test.py

```diff
@@ -0,0 +1,61 @@
+"""Test for Torstens's quorum enumeration"""
+from .quorums import enumerate_quorums
+from .quorum_lachowski import contains_slice
+
+def test_enumerate_quorums():
+    """Test enumerate_quorums() with simple example"""
+    slices_by_node = {
+        1: [{1, 2, 3, 7}],
+        2: [{1, 2, 3, 7}],
+        3: [{1, 2, 3, 7}],
+        4: [{4, 5, 6, 7}],
+        5: [{4, 5, 6, 7}],
+        6: [{4, 5, 6, 7}],
+        7: [{7}],
+    }
+
+    def ex28_fbas(nodes_subset, node) -> bool:
+        return contains_slice(nodes_subset, slices_by_node, node)
+
+    quorums = list(enumerate_quorums(ex28_fbas, {1, 2, 3, 4, 5, 6, 7}))
+    assert set(quorums) == set(
+        [frozenset({7}),
+         frozenset({4, 5, 6, 7}),
+         frozenset({1, 2, 3, 7}),
+         frozenset({1, 2, 3, 4, 5, 6, 7})]
+    )
+
+def test_enumerate_quorums_stellar_core():
+    """Test enumerate_quorums() with stellar core style fbas"""
```

Comment:
It probably makes sense to skip this test by default.

Reply:
I think that's fine :)